app/assets/javascripts/application.js.coffee

```diff
@@ -10,4 +10,13 @@
 #= require spectrum
 #= require_tree ./components
 #= require_tree ./pages
-#= require_self
\ No newline at end of file
+#= require_self
+
+format = (icon) ->
+  originalOption = icon.element
+  '<i class="fa ' + $(originalOption).data('icon') + '"></i> ' + icon.text
+
+$(document).ready ->
```

Comment:
It would be nice to move that code into a new class in `app/assets/javascripts/pages`.